plugin/pkg/scheduler/scheduler.go

```diff
@@ -100,14 +101,15 @@ type Config struct {
 	// question, and the error
 	Error func(*v1.Pod, error)
 
-	// Recorder is the EventRecorder to use
+	// recorder is the EventRecorder to use
 	Recorder record.EventRecorder
 
 	// Close this to shut down the scheduler.
 	StopEverything chan struct{}
 }
 
-// New returns a new scheduler.
+// New returns a new scheduler.  This can give way over time to the "NewFromConfigurator" function,
+// which is interface driven.
```

Comment:
Shouldn't we add a TODO to remove this function and replace its usage with NewFromConfigurator?